hicommonwealth/commonwealth
# Added stakes tab data and column sorting

## Link to Issue
Closes: 

- https://github.com/hicommonwealth/commonwealth/issues/6669
- https://github.com/hicommonwealth/commonwealth/issues/6670
- https://github.com/hicommonwealth/commonwealth/issues/6671
- https://github.com/hicommonwealth/commonwealth/issues/6673

## Description of Changes
- Added table to stakes tab in my stake page
- Added wallet address filtering for data in stake tab
- Added community name search in stake tab
- Added column sorting in stake tab
 
## "How We Fixed It"
N/A

## Test Plan
Pre-req: Connect multiple addresses to a single account in any community

- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` in `.env`
- Visit `/myCommunityStake` -> "Stakes" tab
- Verify you can search communities with name and symbol
- Verify you see multiple addresses in the address filter dropdown (and filter them)
- Verify you can filter the stake table and each column shows correctly filtered values
- Verify you can sort multiple columns using `Shift + Enter/Click` on sort direction button


## Deployment Plan
N/A

## Other Considerations
N/A
